### PR TITLE
test(QTM-327): Migrates unit tests to testing-library/react

### PR DESCRIPTION
**Pull request by toncabral88**
## Description
https://jirasoftware.catho.com.br/browse/QTM-327

## Setup
to view the components behavior, run `yarn storybook`

## Review guide
- [ ] Coverage (coverage status should not regress)\
      - `yarn test:coverage`
- [ ] Unit tests (yarn test:components)
- [ ] Code review

### CODE REVIEW

File: components/GlobalStyle/sub-components/Typography.unit.test.jsx [hunk 1]
```diff
@@ -1,18 +1,17 @@
...
-
-    expect(typography).toMatchSnapshot();
+    expect(render(<Typography />).asFragment()).toMatchSnapshot();
   });
 
   it('Should set standard typeface', () => {
-    renderer.create(<Typography />);
+    render(<Typography />);
 
     expectCSSMatches(`* {
```
Comment (matheusrpp) [inline components/GlobalStyle/sub-components/Typography.unit.test.jsx:16]: Did you gave you on replacing these?

**Review by matheusrpp — APPROVED** [review]
## Review guide
- [x] Coverage (coverage status should not regress)\
      - `yarn test:coverage`
- [x] Unit tests (yarn test:components)
- [x] Code review

File: components/GlobalStyle/sub-components/Typography.unit.test.jsx (hunk 1 above)
Comment (toncabral88) [reply components/GlobalStyle/sub-components/Typography.unit.test.jsx:16]: I don't understand man.

Comment (matheusrpp) [reply components/GlobalStyle/sub-components/Typography.unit.test.jsx:16]: Friday, doing the daily, you commented on replacing these `expectCSSMatches`

**Pull request merged**